src/ReportManager/DetailsSection/index.js

```diff
@@ -10,7 +10,7 @@ import { useTranslation } from 'react-i18next';
 import { ReactComponent as PencilWritingIcon } from '../../common/images/icons/pencil-writing.svg';
 
 import { calcGeometryTypeForReport } from '../../utils/events';
-import { EVENT_FORM_STATES, VALID_EVENT_GEOMETRY_TYPES } from '../../constants';
+import { EFB_SCHEMA_DRAFT, EVENT_FORM_STATES, VALID_EVENT_GEOMETRY_TYPES } from '../../constants';
```

Comment:
`EFB` doesn't belong anywhere in the code here. We're matching a new spec for event type schemas, and the fact that these come from a form builder UI using terminology which is entirely internal to us isn't good naming convention.

Reply:
I agree, but also I wonder, how should we differentiate those two flows then? They read different schemas, they use different components... We sure need to call them differently somehow.

Reply:
This will be coming from `v2` APIs, so the distinction point is from the very beginning. You can key off of the version of the schema, the API endpoint source, where the data comes from in the redux store, etc etc. Tons of good clear indicators of what "version" of the schema you're using, without getting stuck on a weird naming convention

Reply:
Hmm I'm not sure if I get you or if we are not talking about the same thing. I know the data will come from a v2 endpoint but how do I know that when reading this file?

We already have a Form component, Gabo is adding a second one. My question is how can we call the new`<Form />` component in a way that it tells you that it's the new component for the new form. `FormV2`? Hahaha. And even if reading from a new redux store is a way to infer the version of the schema, I don't think that's a good way to expect devs to understand what's going on. I'd rather "scream" with file names, variable names, method names, etc... Making the code tell you what is going on.